docs/rate-limits.md

```diff
@@ -71,8 +71,8 @@ per account, per hostname, per hour. This limit will be higher on staging so you
 can use staging to debug connectivity problems.
 
 The "new-reg", "new-authz" and "new-cert" endpoints have an **Overall
-Requests** limit of 20 per second. All other endpoints have an Overall Requests
-limit of 2000 per second.
+Requests** limit of 20 per second. "/acme", it's other subdirectories and 
```

Comment:
What about: 
> The "/directory" endpoint and the "/acme" directory & subdirectories have a limit of 40 requests per second.

I think putting `/directory` first follows from the other endpoints described in the previous sentence. I also think it's beneficial to remove the second bolded instantiation of the rate limit name since we want each rate limit "kind" to appear once in bold to draw the eye.

What do you think?